src/NotificationEventMailing.php

```diff
@@ -186,7 +186,7 @@ public static function send(array $data)
 
                 $mail->from(new Address($current->fields['sender'], $current->fields['sendername']));
 
-                if ($current->fields['replyto']) {
+                if (!empty($current->fields['replyto'])) {
```

Comment:
don't use empty for testing strings number of chars, use direct comparison ( `!= ''`) or `strlen`.
You may encounter some strings that be considered empty ("false" is primary example)